.gitignore

```diff
@@ -1,3 +1,6 @@
+.gitignore
+GIT_*.cmd
```

Comment:
i'm not familiar with this filetype, but it seems more appropriate in your global gitignore rather than per-project